webcompat/templates/issue.html

```diff
@@ -127,9 +132,17 @@ <h1 class="wc-IssueDetail-title wc-Title--l">
         </div>
     </div>
   </script>
-
+  
+  <script type="text/template" id="qr-image-tmpl">
+    <div class="wc-QrImage-header">
+      <button class="wc-QrImage-btn">Close</button>
+    </div>
+    <div class="wc-QrImage-main"><img src="<%= qrDataUrl %>" alt="QR code"></div>
+  </script>
+  
 {% endblock %}
 {% block extrascripts %}
+<script src="{{ url_for('static', filename='js/vendor/qr.min.js')|bust_cache }}"></script>
```

Comment:
Can we go ahead and add the qr.min.js file to the Grunt concat task?

We talked about lazy-loading this script in the future, but for now I think it's better minified and served with the rest.

See https://github.com/webcompat/webcompat.com/blob/master/grunt-tasks/concat.js#L8.